satellite/payments/stripepayments/stripe.go

```diff
@@ -329,7 +388,33 @@ func (s *service) GetInvoice(ctx context.Context, id []byte) (_ *payments.Invoic
 		CustomFields: customFields,
 		CreatedAt:    time.Unix(inv.Created, 0),
 	}, nil
+}
+
+// GetProjectInvoices returns all invoices for a particular project
+func (s *service) GetProjectInvoices(ctx context.Context, projectID uuid.UUID) (_ []payments.Invoice, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	stamps, err := s.db.ProjectInvoiceStamps().GetAll(ctx, projectID)
+	if err != nil {
+		return nil, stripeErr.Wrap(err)
+	}
+
+	var invoices []payments.Invoice
+	for _, stamp := range stamps {
+		inv, err := s.GetInvoice(ctx, stamp.InvoiceID)
+		if err != nil {
+			return nil, err
+		}
+
+		invoices = append(invoices, *inv)
+	}
+
+	return invoices, nil
+}
 
+// GetProjectInvoiceByStartDate returns invoice invoice for project with specific billing period start date
+func (s *service) GetProjectInvoiceByStartDate(ctx context.Context, projectID uuid.UUID, startDate time.Time) (*payments.Invoice, error) {
+	panic("implement me")
```

Comment:
maybe it would be less aggressive to do something like `return nil, stripeErr.New("unimplemented")`